streaming/fs2/src/main/scala/sttp/openai/streaming/fs2/Fs2OpenAI.scala

```diff
@@ -0,0 +1,48 @@
+package sttp.openai.streaming.fs2
+
+import fs2.{Pipe, RaiseThrowable, Stream}
+import sttp.capabilities.fs2.Fs2Streams
+import sttp.client4.StreamRequest
+import sttp.client4.impl.fs2.Fs2ServerSentEvents
+import sttp.model.sse.ServerSentEvent
+import sttp.openai.OpenAI
+import sttp.openai.OpenAIExceptions.OpenAIException
+import sttp.openai.json.SttpUpickleApiExtension.deserializeJsonSnake
+import sttp.openai.requests.completions.chat.ChatChunkRequestResponseData.ChatChunkResponse
+import sttp.openai.requests.completions.chat.ChatRequestBody.ChatBody
+
+object Fs2OpenAI {
```

Comment:
** suggestion ** How about creating an object called `fs2` in package `sttp.openai.streaming`. This would make the official required import be just a nice `import sttp.openai.streaming.fs2._`.

Reply:
It's problematic, an object named `fs2` breaks imports from actual fs2 library.

Reply:
Imports from `fs2` are only clashing within sttp-openai itself, not for users. You can resolve this by using

```scala
import _root_.fs2.whatever
```
and this is a common pattern.